src/exceptions/OCaml-LGPL-linking-exception.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="OCaml-LGPL-Linking-Exception" name="OCaml LGPL Linking Exception">
+      <crossRefs>
+         <crossRef>https://caml.inria.fr/ocaml/license.en.html</crossRef>
+      </crossRefs>
+      <notes>
+        Adopted by OCaml core in 2001 here: https://github.com/ocaml/ocaml/commit/02ef950033b81fe371759f024faa55f361ba83a6#diff-9879d6db96fd29134fc802214163b95a (git-svn-id: http://caml.inria.fr/svn/ocaml/trunk@4146 f963ae5c-01c2-4b8c-9fe0-0dff7051ff02)
+      </notes>
+    <text>
+      <titleText>
+         <p>OCaml LGPL Linking Exception</p>
+      </titleText>
+      <p>
+        In the following, "the OCaml Core System" refers to all files marked
+        "Copyright INRIA" in this distribution.
+
+        The OCaml Core System is distributed under the terms of the GNU Lesser
+        General Public License (LGPL) version 2.1 (included below).
+
+        As a special exception to the GNU Lesser General Public License, you may
+        link, statically or dynamically, a "work that uses the OCaml Core System"
```

Comment:
To support both the [current][1] "the OCaml Core System" and the [original][2] "the Library", I recommend:

```xml
link, statically or dynamically, a "work that uses <alt name="work1" match=".*">the OCaml Core System</alt>"
```

and similarly below (with `work2`, `work3`, etc.) for all locations of "the OCaml Core System".

[1]: https://caml.inria.fr/ocaml/license.en.html
[2]: https://github.com/ocaml/ocaml/commit/02ef950033b81fe371759f024faa55f361ba83a6#diff-9879d6db96fd29134fc802214163b95aR16